playground/mocks/data/idp/idx/email-challenge-consent.json

```diff
@@ -0,0 +1,71 @@
+{
+  "stateHandle":"02merwUPalfifY9KH-gAqp7P-Qa_gX3LnBm8sxWWWU",
+  "version":"1.0.0",
+  "expiresAt":"2021-03-18T15:29:59.000Z",
+  "intent":"CONSENT",
+  "remediation":{
+    "type":"array",
+    "value":[{
+      "rel":[
+        "create-form"
+      ],
+      "name":"email-challenge-consent",
+      "href":"http://localhost:3000/idp/idx/consent",
+      "method":"POST",
+      "accepts":"application/json; okta-version=1.0.0",
+      "produces":"application/ion+json; okta-version=1.0.0",
+      "requestInfo":[
```

Comment:
Hmm. Not sure, probably because it needs to be displayed to the user, hence they're in `remediation`. May be we can push it in uiSchema and expand our uiSchema transformer to just display the text as they are typed `string`. (It could have come directly in the `value` array instead too)

Reply:
We can do that as well but I was more thinking `requestInfo` as contextual data like other information hence shall sit at top level.